scripts/rook.sh

```diff
@@ -26,7 +26,15 @@ function deploy_rook() {
 		kubectl create -f "${TEMP_DIR}/cluster-test.yaml"
 		rm -rf "${TEMP_DIR}"
 	else
-		kubectl create -f "${ROOK_URL}/cluster-test.yaml"
+		# add "mon_warn_on_pool_no_redundancy = false" to ceph.conf if missing
+		# see https://github.com/rook/rook/pull/5925 for upstream status
+		TEMP_DIR="$(mktemp -d)"
```

Comment:
we can move TEMP_DIR and curl before if check and after else we can do kubectl create and rm -rf as we can avoid duplication inside if and else

Reply:
That can be done, but once rook/rook#5925 is merged (and backported) this should not be needed anymore. Do you still want me to change it?